### DISCUSSION TITLE
Welcome to CovsirPhy Discussions!

### DISCUSSION THREAD

**Discussion by lisphilar** [lisphilar/covid19-sir#395] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jparra0419** [comment]
I am trying to run  the example :   dep_usage_policy.ipynb  and I am getting  the error:
 AttributeError: module 'covid19dh' has no attribute 'cite'.   

I was looking for help, and I  found that the "cite" function must be removed.  If this is the case, I wouldn't know how to do it.   At present I am stacked  and I don't know what to do. Please Help  me.

**Reply by lisphilar** [reply]
Could you confirm you are using the latest verion of CovsirPhy (2.20.2)? I thought we removed 
 'covid19dh.cite' function from Covsirphy codes.

FYI.
dep_usage_policy.ipynb may not be good as a tutorial because 'PolicyMeasures' class was deprecated. I will remove the file from example directly later.

The other files of example directory, including usage_quick.ipynb,will be better. Or, please refer to our documentation.
https://lisphilar.github.io/covid19-sir/

**Reply by jparra0419** [reply]
Thank you lisphilar,

I am using 2.4.0 version.  I didn't know this was an old version. I didn't see anywhere that 2.20.2 was the version without the "cite" function. I will try this new version to finally some results.
thank you again

**Reply by lisphilar** [reply]
Thank you for trying.

Currently, updates are announced via
- release notes https://github.com/lisphilar/covid19-sir/releases and
- my Twitter account https://twitter.com/lisphilar

If you have any ideas to notify users of updates, kindly share them with issues :)

**Reply by jparra0419** [reply]
I followed your advise to use version 2.20.2 of "covsirphy".

I tried: pip install covsirphy --upgrade; also pip install covsirphy==2.20.2
and I got:
ERROR: Could not find a version that satisfies the requirement covsirphy==2.20.2 (from versions: 2.3.0, 2.3.1, 2.3.2, 2.4.0, 2.4.1, 2.4.2, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.5.4, 2.6.0, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.9.0, 2.9.1, 2.10.0, 2.11.0, 2.12.0, 2.13.0, 2.13.1, 2.13.2, 2.13.3, 2.14.0.
Furthermore I tried:
pip install --upgrade git+https://github.com/lisphilar/covid19-sir.git#egg=covsirphy

and I get the same ERROR. I would appreciate your help  ---Thank you

**Reply by lisphilar** [reply]
This will be discussed in new issue #800.
